match-standards-mix01.Rmd

```diff
@@ -97,193 +120,620 @@ data_SP <- filterFile(data, file = which(data$matrix_pol == "Serum_POS"))
 data_SN <- filterFile(data, file = which(data$matrix_pol == "Serum_NEG"))
 ```
 
-## Positive polarity and solved in water samples
+## Water, positive polarity
+
+We first perform the analysis on the samples with the standards solved in pure
+water acquired in positive polarity mode.
+
+### Data pre-processing
 
-### Chromatographic peak detection
+We perform the pre-processing of our data set which consists of the
+chromatographic peak detection followed by a peak refinement step to reduce peak
+detection artifacts, the correspondence analysis to group peaks across samples
+and finally the gap-filling to fill in missing peak data from samples in which
+no chromatographic peak was detected.
+
+Some notes on the settings for the pre-processing:
+
+- The `bw` parameter for the correspondence is larger than usual because adding
+  the standards in higher concentrations caused considerable retention time
+  shifts for some.
+- The `binSize` was also slightly increased to avoid splitting of features.
 
 ```{r peakdetection, eval = !file.exists(paste0(RDATA_PATH, "processed_data.RData"))}
+## Peak detection
 cwp <- CentWaveParam(ppm = 50,
-                     peakwidth = c(2, 20),
+                     peakwidth = c(2, 18),
                      snthresh = 5,
                      mzdiff = 0.001,
                      prefilter = c(4, 300),
                      noise = 100,
                      integrate = 2)
 
 data_WP <- findChromPeaks(data_WP, param = cwp) 
-```
 
-### Refinement
-
-```{r refinement, message = FALSE, warning = FALSE, eval = !file.exists(paste0(RDATA_PATH, "processed_data.RData"))}
-mnp <- MergeNeighboringPeaksParam(expandRt = 2.5, expandMz = 0.001,
+## Peak refinement
+mnp <- MergeNeighboringPeaksParam(expandRt = 3.5, expandMz = 0.001,
                                   minProp = 3/4)
 data_WP <- refineChromPeaks(data_WP, param = mnp)
-```
 
-### Correspondence analysis
+## Alignment
+pdp1 <- PeakDensityParam(sampleGroups = data_WP$matrix_pol, bw = 3,
```

Comment:
there are rt shifts in the data, specifically due to the higher concentration. The increased `bw` should help to still group such shifted peaks into the same feature.